DATA/production/qc-sync/trditstpc.json

```diff
@@ -161,7 +161,7 @@
       "id": "trdall",
       "active": "true",
       "machines": [],
-      "query": "digits:TRD/DIGITS/0;tracklets:TRD/TRACKLETS/0;triggers:TRD/TRKTRGRD/0;rawstats:TRD/RAWSTATS/0;tracks:TRD/MATCH_ITSTPC;trigrectrk:TRD/TRGREC_ITSTPC",
+      "query": "digits:TRD/DIGITS/0;tracklets:TRD/TRACKLETS/0;triggers:TRD/TRKTRGRD/0;rawstats:TRD/RAWSTATS/0;tracks:TRD/MATCH_ITSTPC;trigrectrk:TRD/TRGREC_ITSTPC;phValues:TRD/PULSEHEIGHT",
```

Comment:
@maroozm  almost. Here in this line you define the input needed for the PH track match task only. I believe that only needs the phValues and everything else can be removed

Reply:
ohh. yes. i was thinking about any issue it could create because our code is not yet in any QC update.

Reply:
hmm. done :)